ThreatDragonModels/ImagingApp/ImagingApp.json

```diff
@@ -37,14 +37,242 @@
+              "attrs": {
+                ".element-shape": {
+                  "class": "element-shape hasNoOpenThreats isOutOfScope"
+                },
+                "text": {
+                  "text": "NITF Source"
+                },
+                ".element-text": {
+                  "class": "element-text hasNoOpenThreats isInScope"
+                }
+              }
+            },
+            {
+              "type": "tm.Flow",
+              "smooth": true,
+              "source": {
+                "id": "86f1b5cd-0369-4d0f-b03c-05d06583e742"
+              },
+              "target": {
+                "id": "9f8c41ea-3d16-4ee1-803b-49d8130cc621"
+              },
+              "vertices": [],
+              "id": "eb52cfc4-a660-47ea-8132-12ddbcb18f1b",
+              "labels": [
+                {
+                  "position": 0.5,
+                  "attrs": {
+                    "text": {
+                      "text": "NITF Ingest",
+                      "font-weight": "400",
+                      "font-size": "small"
+                    }
+                  }
+                }
+              ],
+              "z": 12,
+              "hasOpenThreats": true,
+              "threats": [
+                {
+                  "status": "Open",
+                  "severity": "High",
+                  "type": "Denial of service",
+                  "title": "Potential Process Crash Caused by System Overload",
+                  "description": "A large number of NITFs or a smaller number of very large NITFs can be uploaded to be processed concurrently, using all available server heap. Consider out-of-band processing, stream-based processing, and queueing of images to be processed in an on-disk holding area.",
+                  "$$hashKey": "object:497"
+                },
+                {
+                  "status": "Open",
+                  "severity": "High",
+                  "type": "Denial of service",
+                  "title": "Malicious NITF causes process/stack overflow",
+                  "description": "It is theoretically possible to craft a NITF in such a way that it can cause memory or stack overflow issues.\n\nThe likelihood of this is low or nil and no example of such a NITF is known at this time.",
+                  "$$hashKey": "object:531"
+                },
+                {
+                  "status": "Open",
+                  "severity": "High",
+                  "title": "Malicious NITF triggers execution of system code",
+                  "type": "Elevation of privilege",
+                  "description": "It is theoretically possible to craft a NITF in such a way that it can cause third-party image processing libraries to execute arbitrary system code.\n\nThe likelihood of this is low or nil and no example of such a NITF is known at this time.",
```

Comment:
✏️ It seems that the second sentence about the likelihood either
1. doesn't belong or
2. should be an explanation for the "mitigation", if the answer to my above question is yes.